### DISCUSSION TITLE
May I ask how to solve this problem “Failed to get embeddings from sequence pooling type is not set”

### DISCUSSION THREAD

**Discussion by tanxu1213** [ggml-org/llama.cpp#6278] [Q&A]
raise RuntimeError("Failed to get embeddings from sequence pooling type is not set")
RuntimeError: Failed to get embeddings from sequence pooling type is not set

**Comment by Teachh** [comment]
I downgraded the version to llama-cpp-python==0.2.47 with `pip install llama-cpp-python==0.2.47`.
